libs/utils/android/benchmark.py

```diff
@@ -171,6 +175,8 @@ def __init__(self):
         self.log.info('=== TestEnv setup...')
         self.te = TestEnv(self._getBmConf())
         self.target = self.te.target
+        # TODO: Check if TestEnv already provides support for rebooting
+        self._reboot()
```

Comment:
You right, that's not optimal.
I would however suggest to keep the definitions here and allow subclasses to call a reboot when required by a specific test configuration.

This means that, in the provided jankbench example, we will call a reboot just one time before running every required configuration.